main.go

```diff
@@ -1,16 +1,21 @@
 package main
 
-import "fmt"
+import (
+	"fmt"
+
+	"github.com/serenize/snaker"
+	"test-ci/lib"
```

Comment:
__[gometalinter]__
* [gotype] could not import test-ci/lib (can't find import: )
* [aligncheck] could not import test-ci/lib (cannot find package "test-ci/lib" in any of:
* [interfacer] could not import test-ci/lib (cannot find package "test-ci/lib" in any of:
* [unconvert] could not import test-ci/lib (cannot find package "test-ci/lib" in any of:
* [structcheck] unused struct field could not import test-ci/lib (cannot find package "test-ci/lib" in any of:
* [errcheck] error return value not checked (could not import test-ci/lib (cannot find package "test-ci/lib" in any of:)
* [varcheck] unused global variable could not import test-ci/lib (cannot find package "test-ci/lib" in any of:

<!-- Comment by [SideCI](https://sideci.com) -->